src/App.js

```diff
@@ -1,11 +1,69 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./components/pokemoncard/PokemonCard"
+import pokemonlogo from './assets/pokemonlogo.png';
+import Header from "./components/header/Header"
+import Button from "./components/button/Button"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+
+    const [ error, setError ] = useState('');
+    const [ pokemon, setPokemon ] = useState ( [])
+    const [ endpoint, setEndpoint ] = useState(`https://pokeapi.co/api/v2/pokemon`)
+    const [loading, setLoading] = useState(true);
+
+    useEffect (() => {
+        async function getPokemonData() {
+            setLoading(true);
+            try {
+                const result = await axios.get(`${endpoint}`);
```

Comment:
Hier kan het iets simpeler, omdat de string niet anders is dan `endpoint` zelf
```suggestion
                const result = await axios.get(endpoint);
```